src/globus_sdk/_testing/data/search/index_list.py

```diff
@@ -0,0 +1,51 @@
+import uuid
+
+from globus_sdk._testing.models import RegisteredResponse, ResponseSet
+
+INDEX_IDS = [str(uuid.uuid1()), str(uuid.uuid1())]
+SUBSCRIPTIONS = [None, str(uuid.uuid1())]
+
+
+RESPONSES = ResponseSet(
+    metadata={"index_ids": INDEX_IDS, "subscription_ids": SUBSCRIPTIONS},
```

Comment:
The metadata structure here seems disjointed; it seems like it should be a dict of index IDs to subscription IDs, or a tuple, or something that relates these.

```python
INDEX_MAPPING = {
    str(uuid.uuid1()): None,  # First index has no subscription
    str(uuid.uuid1()): str(uuid.uuid1()),  # Second has a subscription
}
```

or

```python
INDICES = (
    (str(uuid.uuid1()), None),
    (str(uuid.uuid1()), str(uuid.uuid1()),
)
```

As it stands, the index ID and subscription ID mappings are independent of each other except by force of will or convention.

Reply:
I hesitated to make these into a dict because it doesn't communicate the ordering as strongly, but I think that might be best.
It's awkward because there's a question of "what data can we provide which doesn't simply recapitulate the same data as is in the response?"

A dict of the following form seems good to me:
```python
index_id0 = str(uuid.uuid1())
index_id1 = str(uuid.uuid1())
INDEX_ATTRIBUTES = {
    index_id0: {"subscription_id": None},
    index_id1: {"subscription_id": str(uuid.uuid1())},
}
```

I'll refactor that shortly.